index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="./style.css">
+  <script src="https://kit.fontawesome.com/66aa7c98b3.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <nav id="navbar">
+    <ul class="nave_link">
+      <li class="link-text"><a href="#welcome-section" class="link">About</a></li>
+      <li class="link-text"><a href="#projects" class="link">projects</a></li>
+      <li class="link-text"><a href="#" class="link">Contact</a></li>
+    </ul>
+  </nav>
+
+  <section id="welcome-section">
+    <div id="text">
+      <h1 class="h1">Hey i am Mimic</h1>
+      <h3 class="h3">a web developer</h3>
+    </div>
+  </section>
+  <section id="projects">
+    <h2 class="project-tile">project-tile</h2>
+    <div class="continer-items">
+      <div class="card">
+        <img src="./low-angle-photography-of-red-concrete-house-3013120.jpg" alt="Denim Jeans">
+        <h1>Tailored Jeans</h1>
+        <p class="price">$19.99</p>
+        <p>Some text about the jeans. Super slim and comfy lorem ipsum lorem jeansum. Lorem jeamsun denim lorem jeansum.
+        </p>
+        <p><button type="button">Add to Cart</button></p>
+      </div>
+      <div class="card">
+        <img src="./low-angle-photography-of-red-concrete-house-3013120.jpg" alt="Denim Jeans">
+        <h1>Tailored Jeans</h1>
+        <p class="price">$19.99</p>
+        <p>Some text about the jeans. Super slim and comfy lorem ipsum lorem jeansum. Lorem jeamsun denim lorem jeansum.
+        </p>
+        <p><button type="button">Add to Cart</button></p>
+      </div>
+      <div class="card">
+        <img src="./low-angle-photography-of-red-concrete-house-3013120.jpg" alt="Denim Jeans">
+        <h1>Tailored Jeans</h1>
+        <p class="price">$19.99</p>
+        <p>Some text about the jeans. Super slim and comfy lorem ipsum lorem jeansum. Lorem jeamsun denim lorem jeansum.
+        </p>
+        <p><button type="button">Add to Cart</button></p>
+      </div>
```

Comment:
**Optional:**
you can use [BEM CSS ](https://css-tricks.com/bem-101/)to write your code, it will improve the way of writing code